cli/src/pcluster/config/validators.py

```diff
@@ -960,6 +960,12 @@ def cluster_validator(section_key, section_label, pcluster_config):
         if max_size < min_size:
             errors.append("max_queue_size must be greater than or equal to initial_queue_size")
 
+    if not section.get_param_value("key_name"):
```

Comment:
I'd define a key_pair validator, it's a self contained validation. To avoid having too many validators in the section.

Reply:
The reason I did it here is because validators are not called on None values.

Reply:
ah, good reason